### PR TITLE
New components 

### PR DESCRIPTION
**Pull request by letsbehuman**
I have created newUserBtn, goBackBtn, cameraBtn and modify CardComponent :D

### CODE REVIEW

File: Front-end/components/goBackbtn.js [hunk 1]
```diff
@@ -1,7 +1,27 @@
-import { TouchableOpacity } from "react-native";
-import { AntDesign } from "@expo/vector-icons";
-import { globalStyles } from "../styles/global";
+import { Ionicons } from "@expo/vector-icons";
```
Comment (7-Dany) [inline Front-end/components/goBackbtn.js:1]: @expo/vector-icons is not installed

File: Front-end/screens/CaptureScreen.js [hunk 2]
```diff
@@ -9,7 +9,8 @@ import {
   Alert,
 } from "react-native";
 import bgImage from "../assets/img/bg40.jpg";
-import GoBackBtn from "../components/goBackbtn";
+import GoBackBtn from "../components/goBackBtn";
```
Comment (7-Dany) [inline Front-end/screens/CaptureScreen.js:12]: goBackBtn it should be goBackbtn

File: Front-end/screens/ScoreScreen.js [hunk 3]
```diff
@@ -12,8 +11,8 @@ import bgImage from "../assets/img/bg40.jpg";
 import { globalStyles } from "../styles/global";
 import ProfileCard from "../components/card";
 import React, { useState, useContext } from "react";
-import ScoreCard from "../components/scoreCard";
-import GoBackBtn from "../components/goBackbtn";
+import Card from "../components/card";
+import GoBackBtn from "../components/goBackBtn";
```
Comment (7-Dany) [inline Front-end/screens/ScoreScreen.js:15]: goBackBtn should be goBackbtn

File: Front-end/screens/ProfileAdmin.js [hunk 4]
```diff
@@ -75,16 +75,14 @@ export default function ProfileAdmin({ navigation }) {
           submitHandle={submitHandle}
           openImagePickerAsync={openImagePickerAsync}
         />
-        <View style={styles.usersList}>
+        <View style={globalStyles.userList}>
+          <Card />
```
Comment (7-Dany) [inline Front-end/screens/ProfileAdmin.js]: Using card component without passing item props

File: Front-end/screens/ProfileAdmin.js [hunk 5]
```diff
@@ -75,16 +75,14 @@ export default function ProfileAdmin({ navigation }) {
...
           <FlatList
             keyExtractor={(user) => user.id}
             horizontal={true}
             data={users}
             renderItem={({ item }) => (
-              <EditProfileCard
-                user={item}
-                deleteUserHandler={deleteUserHandler}
-              />
+              <Card user={item} deleteUserHandler={deleteUserHandler} />
```
Comment (7-Dany) [inline Front-end/screens/ProfileAdmin.js:84]: There is no user and deleteUserHandler in Card props

**Review by 7-Dany — CHANGES_REQUESTED** [review]

File: Front-end/screens/CaptureScreen.js (hunk 2 above)
Comment (letsbehuman) [reply Front-end/screens/CaptureScreen.js:12]: I changed the component name so everything will be ...Btn

**Pull request merged**